src/lib/components/FormChipCell/FormChipCell.js

```diff
@@ -220,16 +223,46 @@ const FormChipCell = ({
   )
 
   const handleToEditMode = useCallback(
-    (event, index) => {
+    (event, chipIndex, keyName) => {
       if (isEditable) {
+        const { clientX: pointerCoordinateX, clientY: pointerCoordinateY } = event
+        let isKeyClicked = false
+        const isClickedInsideElement = (pointerCoordinateX, pointerCoordinateY, element) => {
```

Comment:
Not sure what element is? is it chip itself or input? use proper name inputElement or chipElement